pkg/gevent/request_events.go

```diff
@@ -23,6 +23,7 @@ func (RequestCreated) EventType() string {
 // user's request is approved.
 type RequestApproved struct {
 	Request access.Request `json:"request"`
+	Review  access.Review  `json:"review"`
```

Comment:
I propose changing these to just include the reviewer ID, the reason being that the Reviewer struct is almost entirely a clone of the Request, so you will just be duplicating the data. 

Given that the event type indicate the review response already, e.g approved or declined we don't need to include this extra information in the event body. Same goes for the other events changes below
```suggestion
	ReviewerID string `json:"reviewerId"`
```